libs/expo/betterangels/src/lib/static/enumDisplayMapping.ts

```diff
@@ -60,19 +60,33 @@ export const enumDisplayVeteran: { [key in YesNoPreferNotToSayEnum]: string } =
     [YesNoPreferNotToSayEnum.PreferNotToSay]: 'Prefer not to say',
   };
 
-export const enumDisplayRelevant: { [key in RelationshipTypeEnum]: string } = {
-  [RelationshipTypeEnum.CurrentCaseManager]: 'Current Case Manager',
+export const clientRelationEnumDisplay: Partial<{
+  [key in RelationshipTypeEnum]: string;
+}> = {
   [RelationshipTypeEnum.Aunt]: 'Aunt',
   [RelationshipTypeEnum.Child]: 'Child',
   [RelationshipTypeEnum.Cousin]: 'Cousin',
   [RelationshipTypeEnum.Father]: 'Father',
   [RelationshipTypeEnum.Friend]: 'Friend',
   [RelationshipTypeEnum.Grandparent]: 'Grandparent',
   [RelationshipTypeEnum.Mother]: 'Mother',
-  [RelationshipTypeEnum.Organization]: 'Organization',
-  [RelationshipTypeEnum.PastCaseManager]: 'Past Case Manager',
-  [RelationshipTypeEnum.Pet]: 'Pet',
   [RelationshipTypeEnum.Sibling]: 'Sibling',
   [RelationshipTypeEnum.Uncle]: 'Uncle',
   [RelationshipTypeEnum.Other]: 'Other',
 };
+
+export const clientRelevantContactEnumDisplay: Partial<{
+  [key in RelationshipTypeEnum]: string;
+}> = {
+  [RelationshipTypeEnum.CurrentCaseManager]: 'Current Case Manager',
+  ...clientRelationEnumDisplay,
```

Comment:
```suggestion
  ...clientRelationEnumDisplay,
  [RelationshipTypeEnum.CurrentCaseManager]: 'Current Case Manager',
```

Reply:
we have a requirement Current case manager be always at the top

Reply:
oh, duh 🤦